### PR TITLE
Improve Contact Name matching by finding the absolute largest Address Book DB

### PR DESCRIPTION
**Pull request by alexdanilowicz**
Previously, it seems we assumed the largest address book db was the first one, when in reality the one in sources may be bigger.

### CODE REVIEW

File: app/README.md [hunk 1]
```diff
@@ -49,7 +49,7 @@ To give your Terminal full disk access, go to System Preferences > Security and
 If you see a console error that says the database is malformed, then you need to delete the databases the application reads from (so then it can quickly create them from scratch again).
 
 ```
-cd; rm -ir ./leftonread
```
Comment (mccallw23) [inline app/README.md:52]: wouldn't the removal of this slash imply that leftonread is not a directory?

File: app/src/utils/initUtils/index.ts [hunk 2]
```diff
@@ -82,13 +82,13 @@ export async function coreInit(): Promise<sqlite3.Database> {
     }
     try {
       await addContactNameColumn(lorDB);
-      log.info(`INFO: ContactName column added successully`);
+      log.info(`INFO: ContactName column added successfully`);
```
Comment (mccallw23) [inline app/src/utils/initUtils/index.ts:85]: good catch!

File: app/README.md (hunk 1 above)
Comment (alexdanilowicz) [reply app/README.md:52]: For some reason, it is actually directory, but doesn't have a slash. Very weird.

Comment (mccallw23) [reply app/README.md:52]: Weird..

**Comment by alexdanilowicz** [comment]
Closing in favor of https://github.com/Left-on-Read/leftonread/pull/198

**Pull request closed without merging**